src/components/Dashboard/HalfHourCount/HalfHourCount.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+import store from '../../../store';
+import _ from 'lodash';
+import useInterval from '@use-it/interval';
+import moment from 'moment';
+import {
+  TileContainer,
+  RecentCountTitle,
+  TimeRefresh,
+  CountIncrease,
+} from './style';
+
+const countPast30 = () => {
+  let listOfUsers = store.getState().person.list;
```

Comment:
Explicit read from Redux store, hopefully not an issue.

Reply:
Not an issue as of now. Like the rest of the things that read from the store the same way, this will be subject to change